addons/im_livechat/static/src/legacy/widgets/public_livechat_view/public_livechat_view.xml

```diff
@@ -174,7 +174,7 @@
                         </strong>
                     </t>
 
-                    <span t-if="!message.isVisitorTheAuthor()" class="o_PublicLivechatMessage_headerDatePrefix">-</span> <small class="o_mail_timestamp o_PublicLivechatMessage_headerDate" t-att-title="message.getDate().format(dateFormat)"><t t-esc="message.getTimeElapsed()"/></small>
+                    <span t-if="!message.isVisitorTheAuthor()" class="o_PublicLivechatMessage_headerDatePrefix">-</span> <small class="o_mail_timestamp o_PublicLivechatMessage_headerDate" t-att-title="message.getDate().format(dateFormat)" t-att-data-date="message.getDate().format(dateFormat)"><t t-esc="message.getTimeElapsed()"/></small>
```

Comment:
as an alternative, we could just use "title" attribute in the JS if it's the same data